ProjectLighthouse.Servers.GameServer/Controllers/Login/ClientConfigurationController.cs

```diff
@@ -1,8 +1,10 @@
 #nullable enable
 using System.Diagnostics.CodeAnalysis;
+using System.Text;
 using LBPUnion.ProjectLighthouse.Configuration;
 using LBPUnion.ProjectLighthouse.Database;
 using LBPUnion.ProjectLighthouse.Extensions;
+using LBPUnion.ProjectLighthouse.Serialization;
```

Comment:
Remove unused imports
```suggestion
using LBPUnion.ProjectLighthouse.Configuration;
using LBPUnion.ProjectLighthouse.Database;
using LBPUnion.ProjectLighthouse.Extensions;
```